code/modules/mob/living/carbon/life.dm

```diff
@@ -149,7 +149,7 @@
 
 	else //Enough oxygen
 		failed_last_breath = 0
-		if(oxyloss)
+		if(health >= HEALTH_THRESHOLD_CRIT)

```

Comment:
I mean, it should be doing damage each tick instead of every four, damage can handle decimals

Reply:
The code gets way messier and more complicated if you try and make it do decimals

Reply:
There is already a bizarre check in human lungs that means you take less damage if you're not breathing in crit in an airless area rather than trying to breath in an airless area, I'd rather not add to that awful pile with a check to make you take less damage if you're in soft crit